templates/types/streaming/nextjs/package.json

```diff
@@ -18,6 +18,10 @@
     "@radix-ui/react-select": "^2.1.1",
     "@radix-ui/react-slot": "^1.0.2",
     "@radix-ui/react-tabs": "^1.1.0",
+    "@types/papaparse": "^5.3.15",
+    "@llamaindex/core": "^0.4.5",
```

Comment:
@marcusschiesser I think better to add back the llamaindex/core and llamaindex/workflow packages

Reply:
I moved to dev dependencies